tests/framework/extensions/clusters/clusters.go

```diff
@@ -323,9 +323,6 @@ func NewK3SRKE2ClusterConfig(clusterName, namespace string, clustersConfig *Clus
 	}
 	if clustersConfig.ETCD != nil {
 		etcd = clustersConfig.ETCD
-		if etcd.S3 != nil {
```

Comment:
was this meant to be part of this pr?

Reply:
Yeah, I wrote offline about it. This is a small change that was missed in separate etcd snapshot testing for S3 clusters.

Reply:
Resolved offline.